tests/foreman/ui/test_host.py

```diff
@@ -2343,3 +2343,27 @@ def test_positive_cockpit(session, default_sat):
         ), 'cockpit page shows hostname {} instead of {}'.format(
             hostname_inside_cockpit, default_sat.hostname
         )
+
+
+@pytest.mark.tier4
+def test_positive_read_details_page_from_new_ui(session, module_host_template):
+    """Create new Host and read all its content through details page
+
+    :id: ef0c5942-9049-11ec-8029-98fa9b6ecd5a
+
+    :expectedresults: Host is created and has expected content
+
+    :CaseLevel: System
+    """
+    interface_id = gen_string('alpha')
+    with session:
+        host_name = create_fake_host(session, module_host_template, interface_id)
+        assert session.host_new.search(host_name)[0]['Name'] == host_name
+        values = session.host_new.get_details(host_name)
+        assert values['Overview']['HostStatusCard']['status'] == 'All Statuses are OK'
+        assert (
+            values['Overview']['DetailsCard']['details']['mac_address'] == module_host_template.mac
+        )
+        assert values['Overview']['DetailsCard']['details']['host_owner'] == values['current_user']
+        # BZ 2059298
+        # assert values['Overview']['DetailsCard']['details']['comment'] == 'Host with fake data'
```

Comment:
I think this is a minor assert to hold this PR/test getting merged for! 

I would have used to skip that assert until we have that bug open and get this PR merged:
```
if not bz_bug_is_open(2059298):
    assert values['Overview']['DetailsCard']['details']['comment'] == 'Host with fake data'
```

Reply:
it was discussed with developers, and the result is that the problem should be fixed from our side, so something more dynamic was needed.  I was looking for solutions and I came up with locator `.//div[@class="pf-c-description-list__group"]//dd//descendant::*/text()/..` that is looking at all elements that have descendants with text.